libbeat/common/cli/cli.go

```diff
@@ -3,15 +3,25 @@ package cli
 import (
 	"fmt"
 	"os"
+	"runtime/debug"
 
 	"github.com/spf13/cobra"
 )
 
+func rescue() {
```

Comment:
Maybe rename this helper to `fatal` or something similar? To me rescue indicates something being captured and recovered from.

Reply:
I am OK to change it, I went with rescue to rescue from the panic, but we actually exit. So I will go with exitOnPanic. :)